tests/reconfiguration.py

```diff
@@ -69,11 +69,28 @@ def wait_for_reconfiguration_to_complete(network, timeout=10):
 
 @reqs.description("Adding a valid node without snapshot")
 def test_add_node(network, args):
-    new_node = network.create_node("local://localhost")
+    # Note: 127.0.0.1 is supplied explicitly to avoid having differently
+    # assigned IPs for the interfaces, something which the test infra doesn't
+    # support widely yet.
+    operator_rpc_interface = "operator_rpc_interface"
+    host = "127.0.0.1"
```

Comment:
We really want to avoid hardcoding interfaces that way, because it makes running concurrent tests flakier than they need to be. Can you use `net.expand_localhost()` instead?

Reply:
Done.